File: 01. Iniciante/Ruby/BEE1008.rb
# beecrowd 1008 - Salário
func_id = gets.chomp.to_i
hrs_trab = gets.chomp.to_i
vlr_hr = gets.chomp.to_f
salario = hrs_trab * vlr_hr
puts "NUMBER = #{func_id}"
puts format('SALARY = U$ %.2f', salario)

File: 01. Iniciante/Java/BEE1008.java
// beecrowd 1008 - Salário
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        Integer idFunc = sc.nextInt();
        Integer hrsTrab = sc.nextInt();
        Double vlrHora = sc.nextDouble();
        Double salario = hrsTrab * vlrHora;
        System.out.printf("NUMBER = %d\nSALARY = U$ %.2f\n", idFunc, salario);
        sc.close();
    }
}
